code/game/objects/effects/anomalies.dm

```diff
@@ -184,6 +184,11 @@
 /obj/effect/anomaly/grav/high/proc/setup_grav_field()
 	grav_field = new(src, 7, TRUE, rand(0, 3))
 
+/obj/effect/anomaly/grav/high/detonate()
+	for(var/obj/machinery/gravity_generator/main/the_generator in GLOB.machines)
```

Comment:
You think it would be better to use GLOB.gravity_generators instead?

Reply:
I actually tried using that at first and it simply wouldn't affect the generators.
When I swapped to `GLOB.machines` it worked but I have no idea why `GLOB.gravity_generators` doesn't work.